ui/src/common/DateTimeSelector.tsx

```diff
@@ -9,22 +9,24 @@ interface DateTimeSelectorProps {
     showDate: boolean;
     label: string;
     popoverOpen?: (open: boolean) => void;
+    style?: object;
 }
 
 export const DateTimeSelector: React.FC<DateTimeSelectorProps> = React.memo(
-    ({selectedDate, onSelectDate, showDate, label, popoverOpen = () => {}}) => {
+    ({selectedDate, onSelectDate, showDate, label, popoverOpen = () => {}, style = null}) => {
         const [open, setOpen] = React.useState(false);
         const localeData = moment.localeData();
         const time = localeData.longDateFormat('LT').replace('A', 'a');
         const ampm = time.indexOf('a') !== -1;
         const format = showDate ? localeData.longDateFormat('L') + ' ' + time : time;
+        const width = (showDate ? 185 : 105) + (ampm ? 20 : 0);
 
         return (
             <KeyboardDateTimePicker
-                variant="inline"
+                variant="dialog"
                 InputProps={{disableUnderline: true}}
                 title={selectedDate.format()}
-                style={{width: (showDate ? 185 : 95) + (ampm ? 20 : 0)}}
+                style={{minWidth: width, maxWidth: width, ...style}}
```

Comment:
For what is this minWidth/maxWidth needed, isn't just width enough?

Reply:
I was playing with minWidth and turned out I just wanted width. It has been changed.